README.md

```diff
@@ -100,17 +100,17 @@ You can use this to launch automatically supervised tasks.
     * Kill your application in certain condition or restart you subprocesses whenever a certain condition is met.
 
 ## Guarantees
```

Comment:
Most of the items under this headings don't seem like guarantees to me, e.g. "completely asynchronous system design" is more of a feature that might imply certain guarantees. "Asynchronous program boundaries" also seems entirely like a feature to me, ditto for "dynamic supervision ..."

Reply:
That's very true, maybe we can open a followup issue to discuss it ?